### PR TITLE
Refactored joint_rv_types, add MultivariateNormal and MultivariateLaplace functions

### PR DESCRIPTION
**Pull request by ritesh99rakesh**
#### Brief description of what is fixed or changed
Refactored `stats/joint_rv_types.py`
Added `MultivariateNormal` and `MultivariateLaplace` functions

Ping @Upabjojr @sidhantnagpal 
#### Release Notes

- stats
    - Added `MultivariateNormal` and `MultivariateLaplace` functions in `sympy/stats/joint_rv_types.py`

### CODE REVIEW

File: sympy/stats/joint_rv_types.py [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
-'GeneralizedMultivariateLogGamma',
-'GeneralizedMultivariateLogGammaOmega',
-'Multinomial',
-'MultivariateBeta',
-'MultivariateEwens',
-'MultivariateT',
-'NegativeMultinomial',
-'NormalGamma'
-]
+           'Dirichlet',
```
Comment (czgdp1807) [inline sympy/stats/joint_rv_types.py]: I think this indentation wasn't required. In other modules, AFAIK, there isn't any.

**Comment by czgdp1807** [comment]
AFAICT, there aren't any architectural changes in this PR. Isn't it?

File: sympy/stats/joint_rv_types.py (hunk 1 above)
Comment (ritesh99rakesh) [reply sympy/stats/joint_rv_types.py]: I can remove this.

**Comment by ritesh99rakesh** [comment]
> AFAICT, there aren't any architectural changes in this PR. Isn't it?

No, I have added `MultivariateNormal` and `MultivariateLaplace` functions in `stats/joint_rv_types.py`

**Comment by czgdp1807** [comment]
> No, I have added `MultivariateNormal` and `MultivariateLaplace` functions in `stats/joint_rv_types.py`

In the previous case, i.e., in the master, we can create multivariate Normal random variable via `Normal` constructor in `crv_types.py`, so I don't think these are needed. Can you provide some pros of the `MultivariateNormal` over `Normal`.

**Comment by czgdp1807** [comment]
It is apparent that you have done some changes in code format. Can you add the things to https://github.com/sympy/sympy/blob/master/doc/src/modules/stats.rst so that they are visible at docs.sympy.org?

File: sympy/stats/joint_rv_types.py [hunk 2]
```diff
@@ -55,12 +59,12 @@ def JointRV(symbol, pdf, _set=None):
...
-    #TODO: Add support for sets provided by the user
+    # TODO: Add support for sets provided by the user
     symbol = sympify(symbol)
     syms = list(i for i in pdf.free_symbols if isinstance(i, Indexed)
-        and i.base == IndexedBase(symbol))
-    syms.sort(key = lambda index: index.args[1])
-    _set = S.Reals**len(syms)
+                and i.base == IndexedBase(symbol))
+    syms.sort(key=lambda index: index.args[1])
+    _set = S.Reals ** len(syms)
```
Comment (sidhantnagpal) [inline sympy/stats/joint_rv_types.py]: This was appropriate before.

**Comment by sidhantnagpal** [comment]
I don't think we follow a rule of thumb to add spaces around `/`, `*`, or `**` operators, so I'd hesitate making changes to existing code for that.

File: sympy/stats/crv_types.py [hunk 3]
```diff
@@ -48,21 +48,21 @@
 
 from __future__ import print_function, division
 
-from sympy import (log, sqrt, pi, S, Dummy, Interval, sympify, gamma,
-                   Piecewise, And, Eq, binomial, factorial, Sum, floor, Abs,
-                   Lambda, Basic, lowergamma, erf, erfc, erfi,  erfinv, I,
-                   hyper, uppergamma, sinh, atan, Ne, expint, Integral)
+import random
```
Comment (Upabjojr) [inline sympy/stats/crv_types.py]: was it necessary to change all these lines? it's not very comfortable for reviewing.

File: sympy/stats/joint_rv_types.py [hunk 4]
```diff
@@ -70,35 +74,36 @@ def JointRV(symbol, pdf, _set=None):
...
         mu, sigma = self.mu, self.sigma
         k = len(mu)
         args = ImmutableMatrix(args)
         x = args - mu
-        return  S(1)/sqrt((2*pi)**(k)*det(sigma))*exp(
-            -S(1)/2*x.transpose()*(sigma.inv()*\
-                x))[0]
+        return S(1) / sqrt((2 * pi) ** (k) * det(sigma)) * exp(
+            -S(1) / 2 * x.transpose() * (sigma.inv() *
+                                         x))[0]
```
Comment (Upabjojr) [inline sympy/stats/joint_rv_types.py]: if you want to change the style, maybe you should open a new PR. This edit is useless.

**Comment by Upabjojr** [comment]
@ritesh99rakesh please don't mix style changes and feature additions. It makes reviewing the PR more complicated. I'm closing this PR, please try to add minimal changes when opening a PR, I don't want to see hundreds of line changes.

**Pull request closed without merging**